zowe_conformance/terms_and_conditions.md

```diff
@@ -1,4 +1,24 @@
-# Zowe Conformance Program Terms and Conditions
+# Zowe Comformance Program Terms and Conditions
+
+- [Definitions](#definitions)
+- [Related Documents](#related-documents)
+- [Introduction](#introduction)
+    - [Participant Requirements](#participant-requirements)
+- [Process](#process)
+    - [Identify Offerings](#identify-offerings)
+    - [Apply](#apply)
+    - [Test](#test)
+    - [Submit test results](#submit-test-results)
+    - [Successful review](#successful-review)
+    - [Ongoing conformance](#ongoing-conformance)
+- [Use of the Conformance Marks](#use-of-the-conformance-marks)
+- [Participant Marks](#participant-marks)
+- [Term and Termination](#term-and-termination)
+- [Disclaimer of Warranties](#disclaimer-of-warranties)
+- [Limitation of Liability](#limitation-of-liability)
+- [Indemnification](#indemnification)
+- [Entire Agreement Modifications](#entire-agreement-modifications)
```

Comment:
Minor nit, after "Entire Agreement" can you insert a semicolon before "Modifications"? Just in the displayed text, no worries for the anchor link.

Reply:
Semi colon added for the table of contents